modules/apps/object/object-api/src/main/java/com/liferay/object/system/SystemObjectDefinitionMetadataServicesTracker.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.object.system;
+
+/**
+ * @author Marcela Cunha
+ */
+public interface SystemObjectDefinitionMetadataServicesTracker {
```

Comment:
@marcelabc can you use `SystemObjectDefinitionMetadataTrackerImpl` for this? Thx.